doc/getting_started/installation_linux.rst

```diff
@@ -65,65 +70,77 @@ Install Requirements and Dependencies
 Note that both Ninja and Make are installed with these instructions; you only
 need one.
 
-Ubuntu:
+.. tabs::
 
-.. code-block:: console
+   .. group-tab:: Ubuntu
 
-   sudo apt-get install --no-install-recommends git cmake ninja-build gperf \
-     ccache dfu-util device-tree-compiler wget \
-     python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
-     make gcc gcc-multilib
+      .. code-block:: console
 
-Fedora:
+         sudo apt-get install --no-install-recommends git cmake ninja-build gperf \
```

Comment:
This also will need updates if you remove the extra apt install items from the GSG itself.